velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -189,6 +189,9 @@ int main(int argc, char** argv) {
       "st_xmin",
       "st_ymax",
       "st_ymin",
+      // Skipping until the new signature is merged and released in Presto:
+      // https://github.com/prestodb/presto/pull/25521
+      "xxhash64",
```

Comment:
Should we move it to the list that's only excluded when Presto is used as a source of truth (see line 242)?